templates/common/openstack/files/usr-local-bin-openstack-afterburn-hostname.yaml

```diff
@@ -1,12 +0,0 @@
-mode: 0755
-path: "/usr/local/bin/openstack-afterburn-hostname"
-contents:
-  inline: |
-    #!/bin/bash
-    set -euo pipefail
```

Comment:
Note that we're losing the `set -euo pipefail` bash options, as `/usr/local/bin/mco-hostname` doesn't set any of that. So any error during `/usr/local/bin/mco-hostname --openstack` will likely remain unnoticed.

Reply:
I'll add it to mco-hostname